debian/one-compiler-dev.install

```diff
@@ -2,5 +2,6 @@
 # bin
 usr/bin/circledump usr/share/one/bin/
 usr/bin/circle-tensordump usr/share/one/bin/
+usr/bin/tflchef usr/share/one/tflchef
```

Comment:
Um? I thought you didn't need this...

Reply:
@seanshpark The `tflchef` has released in NPU_Compiler. So I asked him to add this file.

@mhs4670go `usr/share/one/bin`?

Reply:
maybe add to `one-compiler-dev.links` too?